app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    new_comment = current_user.comments.new(
+      text: comment_params,
+      user_id: current_user.id,
+      post_id: @post.id
+    )
+    new_comment.update_comments_counter
+    if new_comment.save
+      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", notice: 'Success!'
+    else
+      render :new, alert: 'Error occured!'
+    end
+  end
+
```

Comment:
- [ ] _New_ action for _comments_ is missing. Kindly ensure to define _new_ action for _comments_ controller, similar to:
```ruby
   def new
     @comment = Comment.new
   end
```